Automattic/wp-calypso
# Add Social Advanced Plan

#### Proposed Changes

This change introduces the new enhanced sharing feature for social and adds the premium yearly and monthly plans to Calypso. It also fixes the naming convention for the 1000 shares feature.

#### Testing Instructions



* Head to http://calypso.localhost:3000/checkout/jetpack/jetpack_social_advanced_yearly to add Jetpack Social Premium Yearly to your shopping cart
* Verify that two variants appear (yearly and monthly)
* Verify that you can switch between the variants
 * Verify that you can complete the purchase using credits or the store sandbox.

Check pricing page in Jetpack Cloud, Advanced should not appear in product list


#### Pre-merge Checklist
